ansible-ipi-install/roles/installer/tasks/main.yml

```diff
@@ -4,6 +4,17 @@
   tags:
   - cleanup
   - getoc
+
+- include_tasks: 15_disconnected_registry_existing.yml
+  when:
+    - disconnected_registry is defined
+    - disconnected_registry == "existing"
+
+- include_tasks: 15_disconnected_registry_create.yml
+  when:
+    - disconnected_registry is defined
+    - disconnected_registry == "create"
+
```

Comment:
these should have tags.
thinking:

tags:
- disconnected_registry
I'm not sure if all your tasks within the file will need the same tag. One way to find out is to run the command

ansible-playbook -i inventory/hosts playbook.yml --list-tasks --list-tags

Reply:
done added in commit